packages/core-ethereum/src/index.ts

```diff
@@ -326,10 +326,10 @@ export default class HoprCoreEthereum extends EventEmitter {
   }
 
   private async redeemTicketsInChannelLoop(channel: ChannelEntry): Promise<void> {
-    const channelId = channel.getId().toHex()
+    const channelId = channel.getId()
     if (!channel.destination.eq(this.getPublicKey())) {
       // delete operation before returning
-      delete this.ticketRedemtionInChannelOperations[channelId]
+      delete this.ticketRedemtionInChannelOperations[channelId.toHex()]
```

Comment:
With L.329, nothing is substantially changed - the `toHex()` is simply moved from L.329 to L.332.
Should I change `ticketRedemtionInChannelOperations` into a Map either way?

Reply:
I'd do so since the only advantage for using `Object` rather than `Map` is using non-string keys - which is no longer the case. Also regarding performance, MDN says

> Performs better in scenarios involving frequent additions and removals of key-value pairs.

which is definitely the case

Reply:
Changed `ticketRedemtionInChannelOperations` type into `Map<string, Promise<void>>`